sdk/web-pubsub/arm-webpubsub/test/webpubsub_examples.spec.ts

```diff
@@ -81,10 +81,6 @@ describe("webPubSub test", () => {
           enabled: "false"
         },
         location,
-        networkACLs: {
```

Comment:
why remove this?

Reply:
it's because: 
```
 RestError: Free tier doesn't support setting network ACLs

```
when calling this api

Reply:
Only free tier doesn't support it. Premium supports it

Reply:
Feels like it's an api layer behavior breaking, since it was supported for free tier before. However, I am not sure how to better describe a breaking changes in api layer like this in rest api specs either with swagger or typespec that one property could be impacted by another property's value.

Reply:
treat it as bugfix.